include/zephyr/dt-bindings/misc/nordic-domain-id-nrf9230.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_MISC_NORDIC_DOMAIN_ID_NRF9280_H_
+
+#define NRF_DOMAIN_ID_APPLICATION 2
+#define NRF_DOMAIN_ID_RADIOCORE   3
+#define NRF_DOMAIN_ID_CELL        4
```

Comment:
Rename to match the spec.
```suggestion
#define NRF_DOMAIN_ID_CELLCORE    4
```